app/classifier/tasks/combo/summary.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import tasks from '..';
+
+const ComboTaskSummary = (props) => {
+  let body = 'No annotations';
+  if (props.annotation.value.length > 0) {
+    body = [];
+    props.annotation.value.map((annotation) => {
+      if (!annotation._key) {
+        annotation._key = Math.random();
+      }
+      const task = props.workflow.tasks[annotation.task];
+      const SummaryComponent = tasks[task.type].Summary;
+      body.push(
+        <div key={annotation._key} className="classification-task-summary">
+          <SummaryComponent task={task} annotation={annotation} onToggle={props.onToggle} />
+        </div>
+      );
+    });
+  }
+  return (
+    <div>
+      {body}
+    </div>
+  );
+};
+
+ComboTaskSummary.propTypes = {
+  annotation: React.PropTypes.shape(
+    { value: React.PropTypes.array }
+  ).isRequired,
```

Comment:
I've found that even with a default prop defined for annotations, higher level classifier components can pass down the annotation as `null`, which will override the default value and break things anyway.

This usually happens on first render, at the beginning of a task, when we are waiting for the annotation to initialise.